helm/harvester/charts/harvester/templates/deployment.yaml

```diff
@@ -84,6 +84,9 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
             - while ! mysqladmin ping -h ${HARVESTER_DB_HOST} -p${MARIADB_PASSWORD} --silent; do sleep 1; done;
+              chmod -R 777 /var/log/panda;
```

Comment:
The same comment as others.

https://github.com/PanDAWMS/panda-k8s/blob/main/helm/harvester/charts/harvester/templates/deployment.yaml#L95

Reply:
removed.